systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -385,6 +397,157 @@ void testCruiseControlMetrics() {
         }
     }
 
+    @Test
+    void testKafkaMetricsSettings() {
+        String expectedMetricsConfig = "{\"lowercaseOutputName\":true,\"rules\":[{\"labels\":{\"clientId\":\"$3\",\"partition\":\"$5\",\"topic\":\"$4\"},\"name\":\"kafka_server_$1_$2\",\"pattern\":\"kafka.server<type=(.+),\n" +
```

Comment:
Wouldn't be better actually check the metrics exported by Kafka instead of just checking the configuration? I think Jakub is proposing the same.